welcome.md

```diff
@@ -0,0 +1,17 @@
+:wave: Hi there, and welcome to the SparkPost community! I'm Sparky, your friendly Slackbot, and I'm here to give you a quick tour so you feel a little more at home.
```

Comment:
let's add a space between "Slack" and "bot" -> "I'm Sparky, your friendly Slack bot,...."

Slackbot is the built in bot in slack :)